lib/custom/src/MShop/Service/Provider/Payment/Stripe.php

```diff
@@ -21,6 +22,101 @@ class Stripe
 	extends \Aimeos\MShop\Service\Provider\Payment\OmniPay
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
+
+	private $beConfig = array(
+		'type' => array(
+			'code' => 'type',
+			'internalcode'=> 'type',
+			'label'=> 'Payment provider type',
+			'type'=> 'string',
+			'internaltype'=> 'string',
+			'default'=> '',
+			'required'=> true,
+		),
+		'onsite' => array(
+			'code' => 'onsite',
+			'internalcode'=> 'onsite',
+			'label'=> 'Collect data locally',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'address' => array(
+			'code' => 'address',
+			'internalcode'=> 'address',
+			'label'=> 'Send address to payment gateway too',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'authorize' => array(
+			'code' => 'authorize',
+			'internalcode'=> 'authorize',
+			'label'=> 'Authorize payments and capture later',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'testmode' => array(
+			'code' => 'testmode',
+			'internalcode'=> 'testmode',
+			'label'=> 'Test mode without payments',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'apiKey' => array(
```

Comment:
For the first 'apiKey' I gave that name because this is how we have to call it when pushing to Omnipay. 
For the second, at Stripe documentation it's called as "Publishable key" http://joxi.ru/12MepDjU4DoEO2
And at the Stripe code examples it don't have specific name, they just insert your key itself

Reply:
OK, in this case can you rename it to "publishableKey" so it's consistent to "apiKey"?

Reply:
Sure